changelog.txt

```diff
@@ -1,5 +1,9 @@
 *** Google for WooCommerce Changelog ***
 
+= 2.9.14 - 2025-06-06 =
+* Tweak - WC 9.9 compatibility.
+* Tweak - WP 6.8 compatibility.
```

Comment:
The "WP 6.8 compatibility" entry is not expected to be added here.
- Compatibility was stated in v2.9.11. This will be duplicated.
   https://github.com/woocommerce/google-listings-and-ads/blob/e0e94236979f2e78ecffa59219516b459a019c88/changelog.txt#L18-L22
- Compatibility stated in v2.9.11 may be removed by the woorelease tool when actually doing release. This will cause the changelog entries to become incorrect.
   ![image](https://github.com/user-attachments/assets/42f58182-6119-45af-904e-53202567d8b3)